src/formatDate.js

```diff
@@ -50,7 +50,47 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat[3];
+  const separatorTo = toFormat[3];
+  const dateValues = date.split(separatorFrom);
+  const newObj = {};
+  let result = '';
+
+  for (let i = 0; i < 3; i++) {
+    let dateName = fromFormat[i];
+    let dateValue = dateValues[i];
+
+    if (dateName === 'YY') {
+      dateName = 'YYYY';
+
+      if (+dateValue < 30) {
```

Comment:
It's better not to use mysterious numbers. would be better to move them to variables (Like 30, 20, 19)